src/components/track-changes/suggestion-list/SuggestionSnippet.tsx

```diff
@@ -42,115 +50,178 @@ export const SuggestionSnippet: React.FC<{ suggestion: TrackedChange }> = ({
     doc: store.doc,
   }))
   const [snippet, setSnippet] = useState<SnippetData | null>(null)
-  const [message, setMessage] = useState('')
   const { dataTracked } = suggestion
+  const contentRef = useRef<HTMLDivElement>(null)
+  useEffect(() => {
+    if (contentRef.current && snippet?.content) {
+      contentRef.current.innerHTML =
+        purify.sanitize(': ' + snippet.content) || ''
+      if (snippet.isEquation) {
+        renderMath(contentRef.current)
+      }
+    }
+  }, [snippet?.content, snippet?.isEquation])
 
   useEffect(() => {
     const getSnippetData = (): {
```

Comment:
looks like this function can accept arguments that it needs to produce a result instead of how it's setup right now and then it can be moved into a different file to make this component more readable.

Reply:
done